app/jinja_filters.py

```diff
@@ -82,7 +81,9 @@ def format_month_year_date(value, date_format='%B %Y'):
 
 @blueprint.app_template_filter()
 def format_datetime(value, date_format='%d %B %Y at %H:%M'):
-    return "<span class='date'>{date}</span>".format(date=datetime.strptime(value, '%Y-%m-%dT%H:%M:%S.%f').strftime(date_format))
+    if datetime.now().hour > datetime.utcnow().hour:
+        return "<span class='date'>{date} GMT(+1)</span>".format(date=datetime.strptime(value, '%Y-%m-%dT%H:%M:%S.%f').strftime(date_format))
+    return "<span class='date'>{date} GMT</span>".format(date=datetime.strptime(value, '%Y-%m-%dT%H:%M:%S.%f').strftime(date_format))
```

Comment:
Too much duplication here. all you really need to do is work out if we are BST and add the `+1`